nrf-softdevice/src/util/portal.rs

```diff
@@ -23,137 +26,144 @@ enum State<T> {
 unsafe impl<T> Send for Portal<T> {}
 unsafe impl<T> Sync for Portal<T> {}
 
+#[cfg(not(feature = "usable-from-interrupts"))]
 fn assert_thread_mode() {
```

Comment:
you can remove this, it's now checked by ThreadModeRawMutex.

Reply:
got it